app/registries/foreman/plugin/rbac_support.rb

```diff
@@ -6,7 +6,7 @@ class RbacSupport
 
       def add_all_permissions_to_default_roles(all_permissions)
         view_permissions = all_permissions.where("name LIKE :name", :name => "view_%")
-        org_admin_permissions = all_permissions.where('resource_type <> :resource_type OR resource_type IS NULL', { :resource_type => "Organization" })
+        org_admin_permissions = all_permissions.merge(Permission.where(resource_type: nil).or(Permission.where.not(resource_type: 'Organization')))
```

Comment:
And this was the second, again merging scopes for correct operator precedence.

Reply:
Just wondering: doesn't `Permission.where.not(resource_type: 'Organization')` include `Permission.where(resource_type: nil)`? I would think `NULL != 'Organization'` but perhaps SQL servers do it different.

Reply:
SQL `NULL` is weird :)

Reply:
I suspected as much.